src/App.js

```diff
@@ -26,22 +27,58 @@ class App extends React.Component {
           <div className="row">
             <div className="col-md-2">
               <Filter />
-              <Basket />
+              <Basket
+                items={this.state.basketItems}
+
+                onDeleteFromBasket = {(deleteItems) => {
+                    const newState = this.state.basketItems.filter((item, index) => index !== deleteItems);
```

Comment:
You can't just delete item from the basket. Each item should have a `count` property which represents how many items would you like to buy. When the user clicks on `delete` it should decrease item's count and, if count is less than 1, remove the whole item from the basket.

Reply:
```suggestion
                    const newItems = // ... some calculations
```
It is not a `newState` in general. You change just items, not a whole state. Variables names should be self explained.